scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
Ummm... I'm getting more lost. 

Passing in index was the only way I could get the slide order number. cp.currentSlideIndex is always set to 0 when go-to-slide.js is run.

Unless we want to update the dom on the fly, but that sounds like a ton of refactoring.